src/test/java/pages/SearchPage.java

```diff
@@ -90,4 +95,18 @@ public void iconEpmtyResults(){
         $(By.cssSelector("div.jira-adbox.jira-adbox-medium.no-results.no-results-message")).shouldBe(Condition.visible);
     }
 
+    public void searchResultsContains(String request){
+        $(".focused").shouldBe(Condition.visible).shouldHave(attribute("data-key")).shouldHave(text(request));
```

Comment:
Can you use XPath instead of css selector? As I understand conditions in .shouldHave() methods can be described in XPath

Reply:
$(".focused") is name of Class so I can change on $(By.className("focused")).
If this is not a solution. Please let me know why xpath much better than className.shouldHave()

Reply:
Maybe something like:
$(By.xpath("//*[@class='focused' and boolean(@data-key)]").shouldBe(Condition.visible).shouldHave(text(request));